src/controllers/lab_test_entry_controller.py

```diff
@@ -73,8 +81,10 @@ def get(self, req: Request, id: str):
             lab_test_entry_dict, code=200, headers={"content-type": "application/json"}
         )
 
+    # this one is for updating bloodSmearURL (and takes multipart-form, we might need another endpoint for updating
```

Comment:
Let's move this to a new controller class and move it to a new route, the update route should stay as it was.